CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ We're following the [Google Go Code Review](https://code.google.com/p/go-wiki/wi
 + Once ready to land your change, squash your commits. Where n is the number of commits in your branch, run
 `git rebase -i HEAD~n`
 
- and subsequently update your remote (you will have to force the push, `git push -f andybons mybranch`). The pull request will update.
+ and subsequently update your remote (you will have to force the push, `git push -f $USER mybranch`). The pull request will update.
```

Comment:
Hey Ben, These instructions didn't work and until I got spencer to approve
me as a developer I was stuck which was frustrating. I'll update the
instructions in another CL and add you as a reviewer.

On Tue, Apr 21, 2015 at 11:48 AM Ben Darnell <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/cockroachdb/cockroach/pull/723#discussion_r28791630>:
>
> > @@ -58,7 +58,7 @@ We're following the [Google Go Code Review](https://code.google.com/p/go-wiki/wi
> >  + Once ready to land your change, squash your commits. Where n is the number of commits in your branch, run
> >  `git rebase -i HEAD~n`
> >
> > - and subsequently update your remote (you will have to force the push, `git push -f andybons mybranch`). The pull request will update.
> > + and subsequently update your remote (you will have to force the push, `git push -f $USER mybranch`). The pull request will update.
>
> On second thought, since this file is for outside contributors we should
> probably rework the instructions to use a fork. (you can't follow these
> instructions until you've been given push access to the main repo). But
> that can be a separate change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/723/files#r28791630>.
>